dydx/client.py

```diff
@@ -61,6 +61,52 @@ def _put(self, *args, **kwargs):
     def _delete(self, *args, **kwargs):
         return self._request('delete', *args, **kwargs)
 
+    def _replace(self, *args, **kwargs):
+        return self._request(self, *args, **kwargs)
```

Comment:
You don't need this as replace uses a `POST` request so you should call `_post()` in your replace function

Reply:
I see you do do that. Just remove this function then